src/Expensify.tsx

```diff
@@ -40,6 +40,8 @@ Onyx.registerLogger(({level, message}) => {
     if (level === 'alert') {
         Log.alert(message);
         console.error(message);
+    } else if (level === 'hmmm') {
+        Log.hmmm(message);
```

Comment:
I think we can get this part merged now - it will start working once the version is bumped. Can you remove the package.json stuff?

Reply:
@marcaaron I reverted but now it throws expected TS compilation problem, because `level` definition comes from `onyx` and current version from `main` doesn't have a new type `hmmm` yet 🤔

Reply:
Ok, I see. Then let's update the type and then we can hold this one some more.

This PR is not too critical since the default is to do an `info` instead of `hmm` so we can just fix the types then come back and bump this one again.